src/main/scala/io/iohk/ethereum/network/p2p/messages/BaseETH6XMessages.scala

```diff
@@ -50,6 +52,24 @@ object BaseETH6XMessages {
 
   }
 
+  implicit val addressCodec: RLPCodec[Address] =
+    implicitly[RLPCodec[Array[Byte]]].xmap(Address(_), _.toArray)
+
+  implicit val accessListItemCodec: RLPCodec[AccessListItem] =
+    RLPCodec.instance[AccessListItem](
+      { case AccessListItem(address, storageKeys) =>
+        RLPList(address, toRlpList(storageKeys))
+      },
+      {
+        case r: RLPList if r.items.isEmpty => AccessListItem(null, List.empty)
+
+        case RLPList(rlpAddress, rlpStorageKeys: RLPList) =>
+          val address = rlpAddress.decodeAs[Address]("address")
+          val storageKeys = fromRlpList[BigInt](rlpStorageKeys).toList
```

Comment:
regarding this BigInt type - were you able to verify if this is good enough? maybe a block from ETC mainnet with a typed transaction and the access list populated?

Reply:
I didn't try to find one, but it feel like a good real-life test case until we connect to mainnet. 
I'll give it a look.

Concerning the StorageKey type, I've seen both BigInt and UInt256 in mantis code, as well as Hash (byte[32]) in go-ethereum. Maybe we should introduce (in another ticket ?) a specific StorageKey type similar to the Address one, with a 32 bytes length